static/css/learnocaml_main.css

```
@@ -224,7 +229,7 @@ body {
     flex-direction: row;
   }
   #learnocaml-main-toolbar > button {
-    border-left: 1px #eee solid !important;
+    border-right: 1px #eee solid !important;
```

Is it really needed?
```suggestion
    border-left: 1px #eee solid !important;
```